frontend/src/app/Settings/FacilityForm.test.tsx

```diff
@@ -594,75 +581,37 @@ describe("FacilityForm", () => {
   });
 
   describe("Device validation", () => {
-    it("warns about missing default device", async () => {
+    it("warns about missing device selection", async () => {
       render(
         <MemoryRouter>
           <FacilityForm
             facility={validFacility}
-            deviceSpecimenTypeOptions={deviceSpecimenTypes}
+            deviceTypes={devices}
             saveFacility={saveFacility}
           />
         </MemoryRouter>
       );
-      // Delete default device
+
+      // Delete devices
       const deleteButtons = await screen.findAllByLabelText("Delete device");
+      expect(deleteButtons).toHaveLength(2);
+      userEvent.click(deleteButtons[0]);
       userEvent.click(deleteButtons[0]);
-      // Attempt save
-      const saveButtons = await screen.findAllByText("Save changes");
-      userEvent.click(saveButtons[0]);
-      await waitFor(async () => expect(saveButtons[0]).toBeEnabled());
-      const warning = await screen.findByText(
-        "A default device must be selected",
-        { exact: false }
-      );
-      expect(warning).toBeInTheDocument();
-    });
-
-    it("properly unsets default device when the default device is changed", async () => {
-      const unusedDevice = { internalId: "device-3", name: "Device 3" };
-
-      render(
-        <MemoryRouter>
-          <FacilityForm
-            facility={validFacility}
-            deviceSpecimenTypeOptions={deviceSpecimenTypes.concat({
-              internalId: "4",
-              deviceType: unusedDevice,
-              specimenType: {
-                internalId: "fake-specimen-id-3",
-                name: "Fake Specimen 3",
-              },
-            })}
-            saveFacility={saveFacility}
-          />
-        </MemoryRouter>
-      );
-      // Change default device
-      const dropdown = screen.getByTestId(
-        "device-dropdown-0"
-      ) as HTMLSelectElement;
 
-      userEvent.selectOptions(dropdown, unusedDevice.internalId);
       expect(
-        (screen.getAllByRole("option", {
-          name: unusedDevice.name,
-        })[0] as HTMLOptionElement).selected
-      ).toBeTruthy();
-
-      const checkboxes = screen.getAllByRole("checkbox");
-
-      checkboxes.forEach((checkbox) => expect(checkbox).not.toBeChecked());
+        await screen.findByText("There are currently no devices", {
+          exact: false,
+        })
+      ).toBeInTheDocument();
```

Comment:
I seriously love how much cleaner this makes the frontend!!

Reply:
@nathancrtr did a great job cleaning up the frontend!